Assets/Scripts/UI/MainMenu/OptionsMenu.cs

```diff
@@ -63,45 +115,94 @@ private void CheckResolutions()
     private void CheckQuality()
     {
         // We invert this list so the dropdown goes from high to low quality.
-        qualityNames = QualitySettings.names;
+        string[] qualityNames = settingsInfo.qualityNames;
         qualityDropdown.AddOptions(qualityNames.Reverse().ToList());
         qualityDropdown.value = QualitySettings.GetQualityLevel() - qualityNames.Length - 1;
         qualityDropdown.RefreshShownValue();
     }
 
-    private float LinearToLogarithmicVolume(float volume)
-    {
-        return 20 * (Mathf.Log10(10 * Mathf.Max(volume, .0001f)) - 1);
-    }
-
     public void SetMasterVolume(float volume)
     {
-        mainAudioMixer.SetFloat(audioGroupMaster, LinearToLogarithmicVolume(volume));
+        settingsInfo.SetMasterVolume(volume);
     }
 
     public void SetMusicVolume(float volume)
     {
-        mainAudioMixer.SetFloat(audioGroupMusic, LinearToLogarithmicVolume(volume) + maxVolumeMusic);
```

Comment:
I'm not sure what the `+ maxVolumeMusic` thing is all about, seems suspicious

Reply:
After discussing with @Fueredoriku, this should be replaced with a constant based on the current max volume